autogluon_zeroshot/contexts/context.py

```diff
@@ -268,9 +268,9 @@ def download(self,
 
     def load(self,
              folds: List[int] = None,
-             load_predictions: bool = False,
+             load_predictions: bool = True,
```

Comment:
I made this change here but we can revert here, the reason is that we use `load_predictions=True` more often.

Reply:
lgtm